lib/modules/manager/bazel/rules/http.ts

```diff
@@ -1,22 +1,38 @@
 import is from '@sindresorhus/is';
 import { z } from 'zod';
-import { regEx } from '../../../../util/regex';
+import { escapeRegExp, regEx } from '../../../../util/regex';
 import { parseUrl } from '../../../../util/url';
 import { GithubReleasesDatasource } from '../../../datasource/github-releases';
 import { GithubTagsDatasource } from '../../../datasource/github-tags';
 import type { PackageDependency } from '../../types';
 import type { UrlParsedResult } from '../types';
 
-export function parseArchiveUrl(
-  urlString: string | undefined | null
-): UrlParsedResult | null {
-  if (!urlString) {
-    return null;
-  }
-  const url = parseUrl(urlString);
-  if (!url || url.host !== 'github.com' || !url.pathname) {
-    return null;
-  }
+const archives = [
```

Comment:
What's the source for this list? I thought Bazel supported `.tar.zst` as well?

Reply:
It's from my head, I just needed to refactor it.
Now I wonder which other cool extensions we now have in 2023.

Reply:
Is this the source? https://bazel.build/rules/lib/repo/http

Reply:
Hah, thanks!